cetmix_tower_server/models/cetmix_tower.py

```diff
@@ -99,3 +104,72 @@ def server_get_variable_value(
         if not value and check_global:
             value = result.get("global")
         return value
+
+    @api.model
+    def server_check_ssh_connection(self, server_reference, attempts=5, timeout=15):
+        """Check if SSH connection to the server is available.
+
+        Args:
+            server_reference (Char): Server reference.
+            attempts (int): Number of attempts to try the connection.
+                Default is 5.
+            timeout (int): Timeout in seconds for each connection attempt.
+                Default is 15 seconds.
+
+        Returns:
+            dict: {
+                "code": int,
+                    # 0 for success,
+                    # 408 if the SSH connection timed out after all attempts,
+                    # 503 if there was a generic SSH connection error.
+                "message": str  # Description of the result
+            }
+        """
```

Comment:
You should add the `Raises` section here which describes exceptions raised by the function. [Example](https://github.com/cetmix/cetmix-tower/blob/41a49cb60f4d38fbcd1008c7a931519863fa7ad4/cetmix_tower_server/models/cx_tower_file.py#L572)